crates/libcontainer/src/syscall/linux.rs

```diff
@@ -141,10 +142,12 @@ impl Syscall for LinuxSyscall {
                         _ => caps::drop(None, CapSet::Bounding, *c)?,
                     }
                 }
-                Ok(())
             }
-            _ => caps::set(None, cset, value),
+            _ => {
+                caps::set(None, cset, value)?;
+            }
```

Comment:
Why do we need this change?

Reply:
because I changed the trait contract from
```
set_capability(&self, cset: CapSet, value: &CapsHashSet) -> Result<(), CapsError>
```
to 
```
set_capability(&self, cset: CapSet, value: &CapsHashSet) -> Result<()>
```
to fulfill anyhow as the Result not using the std Result.